src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="cell cell--start-day-sun cell--month-length-31">
+        <div class="cell__day"></div>
```

Comment:
```suggestion
      <div class="calendar calendar--start-day-sun calendar--month-length-31">
        <div class="calendar__day"></div>
```
it should fix tests. 
now tests can't to find expected selector
![Screenshot from 2024-03-07 17-35-08](https://github.com/mate-academy/layout_calendar/assets/33029864/845bc1c9-1540-49b7-9b29-d46f19767f36)
![Screenshot from 2024-03-07 17-32-33](https://github.com/mate-academy/layout_calendar/assets/33029864/bb83f2c2-65d0-49cf-a2b8-a36865679618)